common/config/victoriametricscloud.go

```diff
@@ -0,0 +1,59 @@
+package config
+
+import (
+	"github.com/odigos-io/odigos/common"
+)
+
+const (
+	VICTORIA_METRICS_ENDPOINT = "VICTORIA_METRICS_ENDPOINT"
+	VICTORIA_METRICS_TOKEN    = "VICTORIA_METRICS_TOKEN"
```

Comment:
Should we also add 'CLOUD` to these? e.g. `VICTORIA_METRICS_CLOUD_ENDPOINT`.

Changing this in the future is a breaking change